docs/installation.rst

```diff
@@ -18,6 +18,9 @@ simple TARDIS calculations.
 
     TARDIS is only compatbile with Python >3.6
 
+.. note::
+    TARDIS only works for Linux and Mac users.
```

Comment:
IMO it can be better rephrased as "Currently, TARDIS is compatible only with LINUX and MacOS."

Reply:
Thanks for the review. Let @wkerzendorf have a say about it. I'll change accordingly.